ci/pipelines/Build.jenkinsfile

```diff
@@ -51,8 +51,11 @@ pipeline
 					{
 						stage("Build ${configurations[i]}")
 						{
-							bat "conan export-pkg ${packageNameInterface}/conanfile.py ${packageNameInterface}/${version}@systelab/${channel} --profile ./package.profile --force"
-							dir("DbAdapterTestUtilities")
+							dir('DbAdapterInterface')
```

Comment:
Why? Are those changes aligned with gitignore and conanfile's export_sources patterns (of each project)?

Reply:
No, this was due to a problem on how the packet was being built. In this case, DbAdapterTestUtilities was being copied inside the packet as well. 

If I call conan inside the folder, only the specific headers of the project are copied into the packet

Reply:
Fixed by modifying how the conanfile is getting the headers